main/src/com/google/refine/commands/GetVersionCommand.java

```diff
@@ -62,10 +62,11 @@ protected class VersionResponse {
         @JsonProperty("java_runtime_name")
         public String java_runtime_name = System.getProperty("java.runtime.name", "?");
         @JsonProperty("java_runtime_version")
-        // public String java_runtime_version = Runtime.getRuntime().version(); // Java 9 or later
-        public String java_runtime_version = System.getProperty("java.runtime.version", "?");
+        public String java_runtime_version = Runtime.getRuntime().version().toString();
         @JsonProperty("display_new_version_notice")
         public String display_new_version_notice = System.getProperty("refine.display.new.version.notice");
+        @JsonProperty("module_names")
```

Comment:
Given that "extensions" are the terminology used in the interface would it make sense to use the same in the API?

Reply:
The `main` module is a Butterfly module but not an extension, I would say, so the current serialization seems to be more faithful to what is being returned.

Reply:
@wetneb, good point.